plugins/modules/site_workflow_manager.py

```diff
@@ -1353,14 +1294,15 @@ def validate_site_input_data(self, config):
                 else:
                     errormsg.append("height should not be None or empty")
 
-                units_of_measure = site.get(site_type, {}).get("units_of_measure")
-                if units_of_measure:
-                    if units_of_measure not in ("feet", "meters"):
-                        errormsg.append(
-                            "units_of_measure: Invalid value '{0}' for units_of_measure in playbook. Must be one of 'feet' or 'meters'.".format(
-                                units_of_measure))
-                else:
-                    errormsg.append("units_of_measure should not be None or empty")
+                if self.compare_dnac_versions(self.get_ccc_version(), "2.3.7.6") >= 0:
+                    units_of_measure = site.get(site_type, {}).get("units_of_measure")
+                    if units_of_measure:
+                        if units_of_measure not in ("feet", "meters"):
+                            errormsg.append(
+                                "units_of_measure: Invalid value '{0}' for units_of_measure in playbook. Must be one of 'feet' or 'meters'.".format(
+                                    units_of_measure))
+                    else:
+                        errormsg.append("units_of_measure should not be None or empty")
```

Comment:
self.log("Missing 'units_of_measure' in floor entry.", "ERROR")

Reply:
Added